### PR TITLE
Replace SlimerJS with PhantomJS

### PR DESCRIPTION
**Pull request by geigerzaehler**
Installation of SlimerJS package keeps failing

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,7 +26,7 @@ lint:
 	eslint lib/ bin/ test/
 
 test:
-	xvfb-run karma start --single-run --browsers SlimerJS --reporters dots
+	karma start --single-run
```
Comment (zeusdeux) [inline Makefile]: 🙇

File: test/unit/entry.spec.js [hunk 2]
```diff
@@ -50,7 +51,7 @@ describe(`createEntry()`, () => {
       })
       it(`got instantiated with its related constructor given field info`, () => {
         Object.getOwnPropertyNames(entry.fields).forEach((fieldId) => {
-          const info = fieldInfo.find((info) => info.id === fieldId)
+          const info = find(fieldInfo, (info) => info.id === fieldId)
```
Comment (zeusdeux) [inline test/unit/entry.spec.js]: Why not stick with `Array.prototype.find`? A whole new dependency just to use `_.find` seems too much imho.

Comment (geigerzaehler) [inline test/unit/entry.spec.js]: Because it is not supported by phantomjs :(

Comment (zeusdeux) [inline test/unit/entry.spec.js]: Shouldn't our transpilation pipeline make that available?

**Pull request merged**